javascript-forms/form-events.js

```diff
@@ -0,0 +1,21 @@
+function handleFocus(event) {
+  console.log('focus fired');
+  console.log(event.target.name);
+}
+
+function handleBlur(event) {
+  console.log('blur fired');
+  console.log(event.target.name);
+}
+
+function handleInput(event) {
+  console.log('eventName:', event.target.name, 'eventValue:', event.target.value);
+}
+
+var focusEvent = document.querySelector('input');
+var blurEvent = document.querySelector('input');
+var inputEvent = document.querySelector('input');
```

Comment:
All three of these are only selecting the first `input` on the page. As of right now, I would imagine that nothing happens when you interact with any of the inputs other than the top one. Again, make sure you are very closely checking that code you submit meets the required functionality.